### DISCUSSION TITLE
Cookie Consent Timeout Value

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1290] [Q&A] [answered]
Where can I set the timeout for the cookie that is issued for  Cookie Consent?

Also I will need to do this on a per portal basis.

The session timeout currently makes that message pop-up  every time you go back to the site

**Comment by meetmandeep** [comment, marked as answer]
@scottw12 There's no UI to change the expiration option for the cookie; however, this is indeed a bug. The cookie should never expire but it's always expiring at end of the session. 

This issue has been resolved in #1295
